R/methods-network.R

```diff
@@ -0,0 +1,82 @@
+# Methods for the Network class
+
+
+# Fancy accessors
+setGeneric("getNodes", function(object) standardGeneric("getNodes"))
+setMethod("getNodes", "Network", function(object) object@nodes)
+setGeneric("getLinks", function(object) standardGeneric("getLinks"))
+setMethod("getLinks", "Network", function(object) object@links)
+setGeneric("getLinkColorScheme", function(object) standardGeneric("getLinkColorScheme"))
+setMethod("getLinkColorScheme", "Network", function(object) object@linkColorScheme)
+# No setters! Once created, a network should only be updated via network methods
+
+# Remove isolated nodes
+# Get isolated nodes
+# Remove redundant links
+# Remove redundant nodes
+# Get Degree list
+# Get Weighted Degree list
+# etc.
+# Threshold network by edge weight
+# Assign color scheme
```

Comment:
Are these the anticipated methods?

Reply:
Oof yes. These are my plans. I had to list them to help me better organize the classes